tests/Command/Dispatcher/DispatchFilesCommandTest.php

```diff
@@ -173,6 +173,11 @@ public static function provideTestFileRenderingCases(): iterable
                         - name: Add PHPUnit matcher
                           run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
 
+                        # until Psalm supports Symfony 8
+                        - name: Remove psalm
+                          if: matrix.symfony-require == '8.0.*'
```

Comment:
There is no symfony 8 here.

Only a `symfony-require: 4.4.*` which seems really old by the way...

Reply:
Oh it's tests